src/vivarium_gates_mncnh/components/delivery_facility.py

```diff
@@ -29,6 +29,11 @@ def setup(self, builder: Builder) -> None:
         self._sim_step_name = builder.time.simulation_event_name()
         self.randomness = builder.randomness.get_stream(self.name)
         self.location = get_location(builder)
+        self.delivery_facility_probabilities = {
```

Comment:
This is properly loading the data from the artifact instead of our constant values